specs/agents/tracing-spans.md

```diff
@@ -54,7 +54,35 @@ The documentation should clarify that spans with `unknown` outcomes are ignored
 
 ### Span stack traces
 
-Spans may have an associated stack trace, in order to locate the associated source code that caused the span to occur. If there are many spans being collected this can cause a significant amount of overhead in the application, due to the capture, rendering, and transmission of potentially large stack traces. It is possible to limit the recording of span stack traces to only spans that are slower than a specified duration, using the config variable `ELASTIC_APM_SPAN_FRAMES_MIN_DURATION`.
+Spans may have an associated stack trace, in order to locate the associated
+source code that caused the span to occur. If there are many spans being
+collected this can cause a significant amount of overhead in the application,
+due to the capture, rendering, and transmission of potentially large stack
+traces. It is possible to limit the recording of span stack traces to only
+spans that are slower than a specified duration, using the config variable
+`ELASTIC_APM_SPAN_STACK_TRACE_MIN_DURATION`. (Previously
+`ELASTIC_APM_SPAN_FRAMES_MIN_DURATION`.)
+
+#### `span_stack_trace_min_duration` configuration
+
+Sets the minimum duration of a span for which stack frames/traces will be
+captured.
+
+This option is case-insensitive.
+
+|                |   |
+|----------------|---|
+| Valid options  | `-1`, `0`, [duration](configuration.md#configuration-value-types) with default suffix `ms` |
```

Comment:
I'd prefer to require the suffix for new options. It's more self-descriptive, especially as currently, different options have different default suffixes. I don't think we need to restrict the range of valid options there, so that `-1ms` and `-1s` are both valid.

Reply:
Are we going to require the suffix for `0` and `-1`? Seems a bit odd, but hard to validate if we only *sometimes* require the suffix I suppose.

Reply:
I agree with both the oddness and the necessity for it.